src/routes/Main/Calendar/Calendar.tsx

```diff
@@ -26,13 +26,15 @@ export const Calendar = () => {
               p={2}
               color="primary.400"
             />
-            Calendar<Badge>Coming soon</Badge>
+            Calendar <Badge>(coming soon)</Badge>
           </Heading>
         </CardHeader>
         <CardBody>
-          Get a bird&apos;s eye view of your timetable, and see what&apos;s
-          coming. View your sport from Clipboard, see when you assignments are
-          due from Canvas and more.
+          Calendar is currently in development. Please check back in a few days.
+          If you have any suggestions, <Link
+            href="https://github.com/debater-coder/timetabl-app/issues/new?assignees=debater-coder&labels=&template=feature_request.md&title=Feature%20Request%20for%20Calendar"
+            color={"primary.500"}
+          isExternal>please make a feature request</Link>.
```

Comment:
This is good, but there should also be some information on what Calendar is. I think the best route is to combine parts of the original text with this text.